### ISSUE TITLE
Using the 'FixedEstimator' behavior model

### ISSUE THREAD

**Issue by turtleMaster20** [tue-robotics/wire#25]
I want to pass the dimensions of detected objects as `FixedEstimator` model. The idea being that they do not affect the data association as the dimensions coming out of my detection are not entirely consistent. So I want to just pass this value. If a state sees a new evidence which is potentially the new state (or influences the new state), then it will update the position (and orientation) based on this this new evidence and the dimensions of detection will be overwritten from the new Evidence.

When I try to load the `FixedEstimator` (even without any parameters, or with parameters from uniform or discrete estimators), it crashes the `wire_core`.

Can you give me any tips regarding how to use this particular behavior model?

Alternatively I was thinking I could still use the `PositionEsitmator` with a very high co-variance for this purpose.

**Comment by MatthijsBurgh** [comment]
@turtleMaster20 Could you please provide any logs of the crash?

@jelfring any documentation on how to the `FixedEstimator`?

**Comment by turtleMaster20** [comment]
> @turtleMaster20 Could you please provide any logs of the crash?
> 
> @jelfring any documentation on how to the `FixedEstimator`?

There is no error message. The node `wire_core` just crashes without any warnings:

```
[wire_server-2] process has died [pid 10078, exit code -11, cmd /home/${user}/catkin_ws/devel/lib/wire_core/wire_server __name:=wire_server __log:=/home/${user}/.ros/log/c05c2fd8-c384-11eb-b2cb-70665563ddcd/wire_server-2.log].
```

**Comment by MatthijsBurgh** [comment]
The pointer logic in the `FixedState` estimator was incorrect. I have fixed this in 9ba8fdce5ead38a1e97d219bac3df425a24c8e67. I haven't checked the other estimators at the moment. But please checkout the `fix/parser` branch and see if that works for you.
